src/index.js

```diff
@@ -1,24 +1,205 @@
 import './style.css';
 
-const mainContainer = document.querySelector('.container');
-const todoTasks = [{
-  description: 'wash the dishes',
-  completed: true,
-  index: 1,
-},
-{
-  description: 'complete to do list project',
-  completed: true,
-  index: 2,
-},
-];
-
-document.addEventListener('DOMContentLoaded', () => {
-  todoTasks.forEach((n) => {
-    const output = document.createElement('div');
-    output.classList.add('listitem');
-    output.setAttribute('id', `${n.index}`);
-    output.innerHTML = `<div><input type="checkbox"> ${n.description}</div> <span class="listitemIcon"><i class="fa-solid fa-ellipsis-vertical"></i></span>`;
-    mainContainer.appendChild(output);
+class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+
+  static task() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return tasks;
+  }
+
+  static addTask(task) {
+    const taskItem = document.querySelector('textarea');
+    const tasks = this.task();
+    tasks.push(task);
+    taskItem.value = '';
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    window.location.reload();
+  }
```

Comment:
- [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
![image](https://user-images.githubusercontent.com/81932478/193042487-72173fcc-9d56-491f-a7c2-c96c5a205933.png)

- So when I add a fifth task it will have an index of four, kindly fix this.

Reply:
![image](https://user-images.githubusercontent.com/34026019/193045470-e00b03be-1eb8-41f7-b719-59d3578593e6.png)
completed is set to false and each task has property called index and for the above example the first task has index 1